src/main/java/stsjorbsmod/campfire/ThirstOption.java

```diff
@@ -0,0 +1,35 @@
+package stsjorbsmod.campfire;
+
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.ui.campfire.AbstractCampfireOption;
+import stsjorbsmod.effects.CampfireThirstEffect;
+
+import static stsjorbsmod.JorbsMod.makeID;
+
+public class ThirstOption extends AbstractCampfireOption {
+    public static final String[] TEXT;
+
+    public ThirstOption(boolean active) {
+        this.label = TEXT[0];
+        this.usable = active;
+        this.updateUsability(active);
+    }
+
+    public void updateUsability(boolean canUse) {
```

Comment:
Should we be making this option unusable if you somehow reach it without having any cards with wrath stacks?